packages/ember-runtime/lib/system/string.js

```diff
@@ -115,6 +115,22 @@ function capitalize(str) {
   return CAPITALIZE_CACHE.get(str);
 }
 
+function deprecateEmberStringUtil(fn, opts = {}) {
```

Comment:
This seems roughly like the same function in the other file, can we share between the two and only have one implementation?